sdk/textanalytics/ai-text-analytics/README.md

```diff
@@ -2,7 +2,7 @@
 
 [Azure TextAnalytics](https://azure.microsoft.com/services/cognitive-services/text-analytics/) is a cloud-based service that provides advanced natural language processing over raw text, and includes six main functions:
 
-__Note:__ This SDK targets Azure Text Analytics service API version 3.0.
+__Note:__ This SDK targets Azure Text Analytics service API version 5.1.0-preview.1.
```

Comment:
```suggestion
__Note:__ This SDK targets Azure Text Analytics service API version 5.1.0-beta.1.
```

Reply:
i thought the sdk version is the one that has beta in it and endpoint version is the one that has preview in it, no?

Reply:
Ohh I see. My bad, here you are talking about the service itself.

Then it should be `3.1-preview.1` for this PR

Reply:
my bad, will fix it.